deepparse/network/bpemb_seq2seq.py

```diff
@@ -71,15 +77,16 @@ def forward(self,
                 `teacher forcing <https://machinelearningmastery.com/teacher-forcing-for-recurrent-neural-networks/>`_.
                 Default value is None since we mostly don't have the target except for retrain.
         Return:
-            The tensor of the address components tags predictions.
+            Either a Tensor of the predicted sequence or a a tuple (``x``, ``y``) where ``x`` is the predicted sequence
+            and ``y`` is the attention weights if attention mechanism is activated.
         """
         batch_size = to_predict.size(0)
 
         embedded_output = self.embedding_network(to_predict, decomposition_lengths)
 
-        decoder_input, decoder_hidden = self._encoder_step(embedded_output, lengths_tensor, batch_size)
+        decoder_input, decoder_hidden, encoder_outputs = self._encoder_step(embedded_output, lengths_tensor, batch_size)
 
-        max_length = lengths_tensor.max().item()
-        prediction_sequence = self._decoder_step(decoder_input, decoder_hidden, target, max_length, batch_size)
+        prediction_sequence, attention_output = self._decoder_step(decoder_input, decoder_hidden, encoder_outputs,
```

Comment:
_decoder_step ne retourne pas un tuple lorsqu'il n'y a pas d'attention donc cette ligne plante (Too many values to unpack). Idem pour FasttextSeq2seq

Reply:
J'ai proposer un fix 05e4b68 dit moi ce que tu en pense.

Reply:
Nice! Ça règle le problème!